PINCache/PINDiskCache.m

```diff
@@ -113,8 +113,8 @@ - (instancetype)initWithName:(NSString *)name rootPath:(NSString *)rootPath
         //this is only safe because we use a dispatch_semaphore as a lock. If we switch to an NSLock or posix locks, this will *no longer be safe*.
```

Comment:
Didn't we switch to an NSLock in a recent pr from @maicki ? Might make sense to update this comment